airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/utils.py

```diff
@@ -0,0 +1,14 @@
+import calendar
+import datetime
+
+
+def datetime_to_secs(dt: datetime.datetime) -> int:
+    return calendar.timegm(dt.utctimetuple())
+
+
+def string_to_date(d: str, f: str = "%Y-%m-%d") -> datetime.date:
+    return datetime.datetime.strptime(d, f).date()
+
+
+def date_to_string(d: datetime.date, f: str = "%Y-%m-%d") -> str:
+    return d.strftime(f)
```

Comment:
nit: we generally use `pendulum` library for datetime operations (it's basically datetime++)

Reply:
Thanks, I will keep it in mind. Should I rewrite these utils for using `pendulum` or they are good as is for just doing formatting?

Reply:
this is fine if it works